payloads/v9/channel.ts

```diff
@@ -122,28 +129,63 @@ export interface APIChannel extends APIPartialChannel {
 	 * See https://discord.com/developers/docs/resources/channel#channel-object-video-quality-modes
 	 */
 	video_quality_mode?: VideoQualityMode;
+}
+
+export interface APIDMChannelBase
+	extends Pick<APITextBasedChannel, 'last_message_id' | 'parent_id'>,
+		Pick<BaseAPIChannel, 'id'> {
 	/**
-	 * The approximate message count of the thread, does not count above 50 even if there are more messages
+	 * The recipients of the DM
+	 *
+	 * See https://discord.com/developers/docs/resources/user#user-object
 	 */
-	message_count?: number;
+	recipients?: APIUser[];
+}
+
+export interface APIDMChannel extends APIDMChannelBase {
+	type: ChannelType.DM;
+}
+
+export interface APIGroupDMChannel extends APIDMChannelBase, APIOwnedChannel, Pick<BaseAPIChannel, 'name'> {
+	type: ChannelType.GroupDM;
 	/**
-	 * The approximate member count of the thread, does not count above 50 even if there are more members
+	 * Application id of the group DM creator if it is bot-created
 	 */
-	member_count?: number;
+	application_id?: Snowflake;
+}
+
+export interface APIThreadChannel extends APITextBasedChannel, APIOwnedChannel {
+	type: ChannelType.GuildPublicThread | ChannelType.GuildPrivateThread | ChannelType.GuildNewsThread;
```

Comment:
Not an issue, I don't even blame you for uniting these into one type, and god I hope they won't make one of the types have extra things or I will scream